packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -136,23 +136,6 @@ abstract class ThemeExtension<T extends ThemeExtension<T>> {
   ThemeExtension<T> lerp(covariant ThemeExtension<T>? other, double t);
 }
 
-// Deriving these values is black magic. The spec claims that pressed buttons
-// have a highlight of 0x66999999, but that's clearly wrong. The videos in the
-// spec show that buttons have a composited highlight of #E1E1E1 on a background
-// of #FAFAFA. Assuming that the highlight really has an opacity of 0x66, we can
-// solve for the actual color of the highlight:
-const Color _kLightThemeHighlightColor = Color(0x66BCBCBC);
-
-// The same video shows the splash compositing to #D7D7D7 on a background of
-// #E1E1E1. Again, assuming the splash has an opacity of 0x66, we can solve for
-// the actual color of the splash:
-const Color _kLightThemeSplashColor = Color(0x66C8C8C8);
-
-// Unfortunately, a similar video isn't available for the dark theme, which
-// means we assume the values in the spec are actually correct.
-const Color _kDarkThemeHighlightColor = Color(0x40CCCCCC);
-const Color _kDarkThemeSplashColor = Color(0x40CCCCCC);
```

Comment:
I think these comments could be very helpful for future reference. Could we possibly revert this change (or find another place to put this explanation)?

Reply:
I believe these were removed for future updates to ThemeData which would deprecate these colors, these shouldn't be anymore anyways.

Reply:
Oh sweet—thanks for the review and the explanation! 🙂